### ISSUE TITLE
Red Nose Day Blurb

### ISSUE THREAD

**Issue by sgorajia** [sgorajia/WorkStuffs#6]
Hi Hubbers!

Join us on **May 19th** as we fundraise to help end child poverty - one nose at a time. 

This year, Red N🔴se Day   is on May 25. Since it falls in the same week as Satellite, we’re going to celebrate early!

_What is Red Nose Day?_
Red Nose Day is a fundraising campaign run by Comic Relief Inc., and brings people together to have fun, raise money and change the lives of kids who need our help the most. The campaign first launched in the UK in 1988, and in the US in 2015. The money raised by Red Nose Day in the US has benefitted programs for children and young people in all 50 states and in 25 countries internationally. 

Beneficiaries for Red Nose Day grants include charity organizations such as Boys & Girls Clubs of America, charity:water , Children’s Health Fund, Feeding America, Gavi, the Vaccine Alliance, National Council of La Raza, Save the Children, and the Global Fund. 

_How Can We Participate?_

Hubbers at HQ:
- Join us on the first floor for Red Nose Day happy hour on Friday 5/19! Get yourself some 🍕  and drinks for $5. 
- Wear your Red Nose, red clothing and take pictures with fun props. Tag your 📷  with #GitHubRedNoseDay and #RedNoseDay on May 25!
- We are also hosting a bake sale/bake off. Individuals/teams that raise the most 💵  will win a prize! Please get in touch with me if you’d like to participate 🍪 🍰. 

All Hubbers: 
- If you’re a remote Hubber in the US, you can purchase a Red Nose from any Walgreens or Duane Reade location! Purchase yours, wear red, and post to the Red Nose Day [Repo](https://github.com/github/RedNoseDay). Non-US Hubbers, wear red, and join us on the Red Nose Day Repo! 
- Consider also donating to the cause directly on GitHub’s team page, [here](https://rednosedayusa.funraise.org/fundraiser/github)!

All proceeds go directly to the [Red Nose Day Organization](https://rednoseday.org/).

**Comment by sgorajia** [comment]
TL:DR

![](https://media.giphy.com/media/WoWm8YzFQJg5i/giphy.gif)


Here is some additional context on Red Nose Day, and what we're doing to celebrate!

What: [Red Nose Day ](https://rednoseday.org/)Happy Hour Fundraiser & Contests
When: Friday, May 19 -- 3p - 5p
Where: HQ 1st Floor + Remotely
About The Cause: To raise money to support projects that ensure kids are safe, healthy, educated and empowered, in all 50 states, and some of the poorest communities in Latin America, Asia and Africa. More details below. 

How you can get involved: 
- Donate to Red Nose Day via Bright Funds  or by cash/check donation at the event on 5/19. Donations will be collected until the official national Red Nose Day on 5/25. 
- Sport a Red Nose (available with donation on 5/19)
- Dress yourself and your pet in red!

Red Noses will be available to anyone who: 
- Can show @sgorajia a screenshot or confirmation of their BrightFunds donation ([US Chapter](https://github.brightfunds.org/organizations/comic-relief-inc-2) or [UK Chapter](https://github.brightfunds.org/organizations/comic-relief))
- Has a cash/check donation 

Red bandanas will also be available for our office pups to sport!

![](https://media.giphy.com/media/8CBEARJ5nTtvO/giphy.gif)

----
Contest Details:

Join us on Friday 5/19 as we celebrate National Red Nose Day (a week early)! 

Themed drinks, fun contests, great eats, and a great cause come together for a fun afternoon! As a part of the festivities, we’ll include a Baking and Red Nose Selfie Contest with prizes! Details here:

Contests:

Best Selfie: 
At HQ: Use the set-up selfie station and it’s red props to get inspired!
Remote: Get creative with your own red props and clothing -- include pets and your families too!  
All: Post your photos to the Red Nose Day [Repo](https://github.com/github/RedNoseDay) by Noon PST on Thurs, May 25th.
Winning selfie will be announced by EOD on May 25th!

Carbs For A Cause (Bake Off): Compete against your co-workers for the best baked good. Winners will be based on best tasting or most creative bakes. 
![](https://media.giphy.com/media/26BoDuPW9bJjGkMRa/giphy.gif)

- Here are the [rules](https://docs.google.com/a/github.com/document/d/1czYbm_0qqg1RK5W0iBVb5HApRo620G4MbJOOo60XVFo/edit?usp=sharing).
Your fellow Hubbbers will vote for their favorite recipes. 
- Hubbers in Boulder, Tokyo & Amsterdam, join in and host your own baking competitions! Choose a winner from your office and send @sgorajia that winner’s name by 5/24.
- Overall, 3 winners from all contests (combined) will be selected by random drawing for one of three prizes!* Winners will be announced by EOD on 5/25.
*Winners may also opt to donate their winnings to Red Nose Day organization. 

![](https://media.giphy.com/media/QfAg6OLtlBeTK/giphy.gif)

**Comment by sgorajia** [comment]
Happy Red Nose Day! 

Thank you to all of you wonderful Hubbers that joined us at the happy hour last Friday, and those of you that donated to Red Nose Day through BrightFunds and at the event! 

As promised, the winner of our Bake Off is --- _drum roll please_ --- @ sambong! Congrats! 😃 
Thank you to all the participants! For those who weren't able to participate this time, keep an eye out for another baking contest this summer!

Congratulations @ mslilylove & @ kmchef on being the winners of our Red Nose Selfie Contest! Your submissions were awesome! 
![img_8832](https://cloud.githubusercontent.com/assets/19175961/26325895/af49f858-3eed-11e7-8ecb-656d4bd08197.JPG)

![img_5390](https://cloud.githubusercontent.com/assets/22753426/26281058/c1c1b6a0-3db8-11e7-8d67-458156969059.JPG)

If you didn't get a chance to (and would still like to) donate to, you still can through [BrightFunds](https://github.brightfunds.org/organizations/comic-relief-inc-2)!

**Comment by msbise** [comment]
I'm going to say take out "Thank you, @msbise for sharing your wonderful snickerdoodles with us!"  And instead add:  "Thank you to all our participants and keep an eye out for an encore baking contest this summer for those who weren't able to compete this time around."

Then use the winning photos in instead of the gif?

And remove "I loved your submissions" and make it more general, like: "Your submissions were awesome!" Something less personal to you.

**Comment by msbise** [comment]
Oh, and also acknowledge that today is actually red nose day?
